src/ui/units/editor/configs/chart/chart-editor-types.ts

```diff
@@ -640,6 +640,15 @@ export function getChartEditorTypes(type: string) {
         },
     } as Record<string, ChartEditorType>;
 
+    if (Utils.isEnabledFeature(Feature.ChartActions)) {
+        chartEditorTypes[EDITOR_TYPE.CONTROL_NODE].tabs.push({
+            name: 'Actions',
```

Comment:
If a user wants to show more than one toast, they will be prevented from doing so by the same values of the name field. Is this expected?